src/Qwiq.Core/IFieldDefinition.cs

```diff
@@ -2,10 +2,10 @@
 
 namespace Microsoft.Qwiq
 {
-    public interface IFieldDefinition : IIdentifiable<int>
+    public interface IFieldDefinition : IIdentifiable<int>, INamed
     {
         [NotNull]
-        string Name { get; }
+        new string Name { get; }
```

Comment:
Why is this newed? Shouldn't it be removed in favor of the name from INamed?

Reply:
@pelavall I'm looking into this now. I think it's a :bug:

Reply:
@pelavall This is to enforce the `[NotNull]` attribute on the field definition--`INamed` does not have that requirement

Reply:
@pelavall This has been removed in d39d2f1.  The null condition is guarded on the `FieldDefinition` class in Core, so it's really a non-issue.